repository/p.json

```diff
@@ -2944,6 +2944,20 @@
 				}
 			]
 		},
+		{
+			"name": "Pyisort",
+			"details": "https://github.com/returnchung/pyisort",
+			"issues": "https://github.com/returnchung/pyisort/issues",
+			"author": ["returnchung"],
+			"labels": ["formatting"],
```

Comment:
The author, issues, and platforms entries can be removed as they're just the defaults.

Reply:
+2950